### ISSUE TITLE
[FEEDBACK] Change "All data returned from APIs must be JSON serializable."

### ISSUE THREAD

**Issue by lorddaedra** [phalt/django-api-domains#8]
Sometimes in russian Telegram chats I read "Django was sexy in 2008-2012, but it looks legacy and outdated in 2019 now...". Thank you for share your opinion, I completely agree with your suggestions. This is the best way to answer them "No, Django is modern framework for modern web. And there is the Guide how to cook Django. How to make Django Great Again".

Okay... 

About APIs. Check this project. https://github.com/gluk-w/django-grpc/tree/master/tests Author tried to implement gRPC server with django project. Also https://github.com/grpc/grpc-web is stable now... So in my opinion line "All data returned from APIs must be JSON serializable." is incorrect. It's depends on API types...

**Comment by phalt** [comment]
@lorddaedra Hey Daedra, thanks for your feedback! I agree that the ruling `All data returned from APIs must be JSON serializable` doesn't accommodate gRPC. Let's look back at what my original goal was to help us figure out a better definition:

In order to reduce cross-domain dependency, I wanted to prevent Objects from one domain entering another, thus I wrote the ruling about JSON serializable. What about this as a better re-wording:

```
All data returned from APIs but be serializable
```

Or how about

```
APIs must never return rich objects from a domain
```

**Comment by phalt** [comment]
Resolved with commit ea36acd08e478e812d0790d8f49efe88a5ab9575
